opea-comps/mega-service/example/orchestrator.py

```
@@ -0,0 +1,386 @@
+        return self
+
+    def flow_to(self, from_service, to_service):
+        try:
+            self.add_edge(from_service.name, to_service.name)
+            return True
+        except Exception as e:
+            logger.error(e)
+            return False
+
+    @opea_telemetry
+    async def schedule(self, initial_inputs: Dict | BaseModel, llm_parameters: LLMParams = LLMParams(), **kwargs):
```

Ruff (0.8.2)

Do not perform function call `LLMParams` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)